riva/clients/asr/client_call.cc

```diff
@@ -33,8 +33,11 @@ ClientCall::AppendResult(const nr_asr::StreamingRecognitionResult& result)
     }
     if (word_time_offsets_) {
       if (num_alternatives > 0) {
-        for (int w = 0; w < result.alternatives(0).words_size(); ++w) {
-          latest_result_.final_time_stamps.emplace_back(result.alternatives(0).words(w));
+        for (int a = 0; a < num_alternatives; ++a) {
```

Comment:
Might be better to do ` for (size_t a = 0` as opposed to int in all loops.

Reply:
I think the reason to use `int` is that the generated protobuf code for size methods return `int` (e.g. `results.alternatives(0).words_size()`).  The important thing I think is that we are not comparing a signed int with an unsigned int.